libimage/runtime.go

```diff
@@ -408,17 +409,15 @@ func (r *Runtime) lookupImageInDigestsAndRepoTags(name string, options *LookupIm
 				}
 			}
 		}
-
-		return nil, "", errors.Wrap(storage.ErrImageUnknown, name)
```

Comment:
If the digest was specified and not found, can the other part succeed?

OTOH see above about code structure, consolidating ErrImageUnknown to a single place might still be better.

Reply:
Previously yes but it was unintentional.  Now with the "latest" enforcement, it cannot anymore.

Reply:
Ah, I didn’t realize the name@digest values would look up name:$any in the other loop.